content/cli/maint_cli/certificates.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: bt_wiki
+title: certificates
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+aliases: /cli/certificates/
+---
+
+{{%children style="h3" description="true"%}}
+
+The `cfy certificates` commands are used to handle procedures related to certificates. 
+
+## Commands
+
+### Replacing Certificates
+**Use case:** Whe the certificates on the cluster instances or on an AIO manager are about to expire and need to be replaced.  
+This feature includes two steps: 
+1. Generating the replace-certificates configuration file. This file should be filled with the new certificates' paths.
+2. Replacing the certificates based on the paths specified in the configuration file.
+
+{{% note %}}
+Replacing certificates can only be done **before** the certificates are expired.
+{{% /note %}}
+
+
+#### generate-replace-config
+
+##### Usage
+
+`cfy certificates generate-replace-config`
+
+Generates the replace-certificates configuration file. Please fill in the generated file with the new 
+certificates' paths and save it.
+ 
+* In case you are using a Cloudify cluster, the cluster instances' private IPs will be filled automatically.
+If your CLI is on a host that is not part of the cluster network, you can specify the instances' public IPs instead.
+* The generated configuration file will be different depending on if you use an AIO manager or a Cloudify cluster. 
```

Comment:
depending on the deployment topology - all-in-one manager or management cluster.